armi/cases/tests/test_cases.py

```diff
@@ -272,20 +277,159 @@ def test_writeInput(self):
             self.assertTrue(os.path.exists(cs["shuffleLogic"]))
 
 
+class MultiFilesInterfaces(interfaces.Interface):
+    """
+    A mostly meaningless little test interface, just to prove that TODO
```

Comment:
Damn. Sorry, I put a TODO in your code. My bad.

```
A little test interface that adds a setting that we need to test copyInterfaceInputs
with multiple files.
```

Reply:
My fault!

Reply:
Good catch! on it